frontend/packages/knative-plugin/integration-tests/features/managed-kafka connection/kafka-source-connection-to-kafka-connection-instance.feature

```diff
@@ -0,0 +1,59 @@
+
+
+        @regression
+        Scenario: Creating Kafka source connecting Kafka Connection and knative service
+            Given user has created knative service "hello-openshift1"
+              And user has created Kafka Connection "kafka-instance-123"
+              And user is at Add page
+             When user clicks Event Source card
+              And user clicks on Kafka Source card
+              And user enters value of Kaka Connection name "kafka-instance-123" in Bootstrap servers
+              And user enters value "strimzi-canary" in Topics
+              And user enters value "foobar" in Consumer group section
+              And user checks the Enable checkbox under SASL
+              And user selects resource "rh-cloud--services-service-account" under User
+              And user selects key as "client-id" under User
+              And user selects resource "rh-cloud--services-service-account" under Password
+              And user selects key as "client-secret" under Password
+              And user checks the Enable checkbox under TLS
+              And user selects Resource under SinK as "hello-openshift1"
+              And user selects Application group as "No Application group"
+              And user enters Application Name as "kafka-source-1"
+              And user clicks on Create button
+             Then user will be redirected to Topology page
+              And user is able to see connection between Kafka Connection "kafka-instance-123" and knative service "hello-openshift1" from kafka source "kafka-source-1"
+
+
+        @regression
+        Scenario: Sidebar of the Kafka connector
+            Given user has created knative service "hello-openshift1"
+              And user has created Kafka Connections "kafka-instance-123"
+              And user has created connection between Kafka Connection "kafka-instance-123" and knative service "hello-openshift1" from kafka source "kafka-source-1"
+              And user is at Topology page
+             When user clicks on the connector connecting to Kafka Connection "kafka-instance-123" from kafka source "kafka-source-1"
+             Then user will see kafka source "kafka-source-1" and Kafka Connection "kafka-instance-123" under Connection in Resource tab of sidebar
+
+
+        @regression, @manual
+        Scenario: Joining the Kafka source with arrow to different Kafka Connections
+            Given user has created knative service "hello-openshift1"
+              And user has created Kafka Connections "kafka-instance-abcd" and "kafka-instance-123"
+              And user has created connection between Kafka Connection "kafka-instance-123" and knative service "hello-openshift1" from kafka source "kafka-source-1"
+              And user is at Topology page
+             When user drags the arrow connecting to Kafka Connection "kafka-instance-123" from kafka source "kafka-source-1"
+              And user drops the arrow to Kafka connection "kafka-instance-abcd"
```

Comment:
can make use of the word `connector` over `arrow`

Reply:
Will replace it with connector